paths/quality_performance_score/index.yaml

```diff
@@ -0,0 +1,71 @@
+---
+summary: Get project's translations' quality performance scores
+description: Get project's translations' quality performance scores
+operationId: projects/quality_performance_score
+tags:
+- Quality performance score
+parameters:
+- "$ref": "../../parameters.yaml#/X-PhraseApp-OTP"
+- "$ref": "../../parameters.yaml#/id"
```

Comment:
`@project = ProjectRepository.user_project_by_code!(current_user, params[:id])` it's defined as `id` on the controller so it has to be `id`. The `id` is just last `id` of the path, whatever it might be.

Reply:
Ah yeah I remember that, like `"/projects/{project_id}/orders/{id}/confirm"` -> order id uses `id`

Reply:
Or does it?